tests/Message/RefundRequestTest.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+namespace Omnipay\PaynlV3\Test\Message;
+
+
+use Omnipay\PaynlV3\Message\Request\RefundRequest;
+use Omnipay\PaynlV3\Message\Response\RefundResponse;
+use Omnipay\Tests\TestCase;
+
+class RefundRequestTest extends TestCase
+{
+    /**
+     * @var RefundRequest
+     */
+    protected $request;
+
+    public function testSendSuccess()
+    {
+        $this->setMockHttpResponse('RefundRequestSuccess.txt');
+
+        $transactionId = '52200435050X2314';
+
+        $this->request->setTransactionReference($transactionId);
+
+        $response = $this->request->send();
+
+        $this->assertInstanceOf(RefundResponse::class, $response);
+        $this->assertEquals($transactionId, $response->getTransactionReference());
```

Comment:
Iam wondering if the 'refund' has a new transactionReference (we have now through API)

Reply:
this should be `EX-9167-1816-4111` in the testcase not 52200435050X2314